splice_machine/src/test/java/com/splicemachine/test/Benchmark.java

```diff
@@ -146,4 +148,28 @@ public static void runBenchmark(int numThreads, Runnable runnable) {
         }
         resetStats();
     }
+
+    public static void rowContainsQuery(int[] levels, String query, Connection conn, String[]... contains) throws Exception {
```

Comment:
I find the method name and signature somehow confusing, is it possible to change it to something like this:

```

/***
 * Executes `query` using `conn` and verifies `matchLines` against the query's result set.
 * 
 * @param matchLines a map of the expected String occurance(s) in each 
 * line of the result set, e.g. { {1,['abc','def']}, {10, 'ghi'}} expects 'abc'
 * and 'def' top appear at line 1 and 'ghi' at line 10.
 */
public static void executeQueryAndMatchLines(Connection conn,
                                             String query,
                                             Map<Integer,String[]> matchLines);
```

Reply:
Sure, I'm fine with either way. The method was copied from our unit test framework. But we can improve its interface for benchmarks.